stdlib/public/core/HashTable.swift

```diff
@@ -102,7 +102,7 @@ extension _HashTable {
   }
 
   internal static func hashSeed(
-    for object: AnyObject,
+    for object: Builtin.NativeObject,
```

Comment:
What's the purpose of this change? (This isn't an objection, I'm just curious.)

Reply:
`AnyObject` is an protocol, and `object: AnyObject` is an existential, and existentials are disallowed in embedded Swift.